Does not support new corejs@3

## Do you want to request a feature or report a bug?
Bug
## What is the current behavior?
When compiling with .babelrc of 
```json
"presets": [
    "@babel/preset-react", 
    ["@babel/preset-env", 
      {
        "useBuiltIns": "usage",
        "corejs": "core-js@3",
        "targets": { "browsers": "> 10%" },
        "modules": false 
      }
    ]
  ],
  "plugins": ["@babel/plugin-syntax-dynamic-import", "@babel/plugin-transform-runtime"],
```
and core-js@3, the following output is produced
```
ERROR in ./node_modules/recharts/es6/polyfill.js
Module not found: Error: Can't resolve 'core-js/es6/math' in '${PATH}/node_modules/recharts/es6'
 @ ./node_modules/recharts/es6/polyfill.js 1:0-26
 @ ./node_modules/recharts/es6/index.js

ERROR in ./node_modules/recharts/es6/polyfill.js
Module not found: Error: Can't resolve 'core-js/es6/number' in '${PATH}/node_modules/recharts/es6'
 @ ./node_modules/recharts/es6/polyfill.js 2:0-28
 @ ./node_modules/recharts/es6/index.js

```
## If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net or similar (template: http://jsfiddle.net/ndLhnegs/).

## What is the expected behavior?

## Which versions of Recharts, and which browser / OS are affected by this issue? Did this work in previous versions of Recharts?
Recharts v1.5.0

--- a/package.json
+++ b/package.json
@@ -55,7 +55,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "core-js": "^2.6.10",
+    "core-js": "^3.4.2",
     "d3-interpolate": "^1.3.0",
     "d3-scale": "^2.1.0",
     "d3-shape": "^1.2.0",

--- a/src/polyfill.js
+++ b/src/polyfill.js
@@ -1,5 +1,5 @@
-import 'core-js/es6/math';
-import 'core-js/es6/number';
+import 'core-js/es/math';
+import 'core-js/es/number';
 /* eslint no-proto: 0 */
 const testObject = {};
 

